### PR TITLE
Add citation

### PR DESCRIPTION
**Pull request by lunson**
As per https://trello.com/c/DR1L8nDA/100-add-citations

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [WWW-merge#187](https://ci.openmicroscopy.org/job/WWW-merge/187/). See the [console output](https://ci.openmicroscopy.org/job/WWW-merge/187/consoleText) for more details.
Possible conflicts:
  - PR #103 lunson 'Sec vul template'
    - css/openmicroscopy.css

**Comment by sbesson** [comment]
Now deployed as https://snoopycrimecop.github.io/www.openmicroscopy.org/citing-us/

**Comment by sbesson** [comment]
The page renders as expected and includes the main publications of the open-source organization for each category. I would think the remaining discussion remains the naming of the page. Alternate possibilities could be `http://www.openmicroscopy.org/citing-ome` or simply `http://www.openmicroscopy.org/citing`.

File: css/openmicroscopy.css [hunk 1]
```diff
@@ -591,6 +591,16 @@ p.job-title {
...
+    border-radius: 2px;
+    background-color: #e3f2fd;
+    font-style: italic;
+    margin-bottom: 10px;
+    padding: 10px 15px;
+}
+
+a.pmid::before { content: "PMID: "; }
+
 /* ========== page: Community ==========*/
```
Comment (sbesson) [inline css/openmicroscopy.css]: @lunson: the only reason for which this PR conflicts with #103 is that the former PR modifies the line underneath. Would it be possible to move the added items right under the `section: About` line? That way I would expect both set of CSS changes would merge cleanly and we could deploy both the citation and the security changes on staging.

**Comment by lunson** [comment]
I am fine with all the naming choices: citing-us, citing-ome, citing. Sorry, I do not see any advantage one might have over the others.

File: citing-us/index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Citing OME
```
Comment (hflynn) [inline citing-us/index.html]: Whichever URL and menu title we pick, it needs to be consistent across the board - at the moment we have 'citing OME' here and 'citing Us' everywhere else

Comment (hflynn) [inline citing-us/index.html]: Looking at the other menu items eg. OME Teams, I think I favour 'Citing OME'

File: citing-us/index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        <!-- end citation-->
+        
+        <!-- begin citation--> 
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-12 medium-4 columns">
+                <h3>Bio-Formats</h3>
+            </div>
+            <div class="small-12 medium-8 columns">
+                <p class="citation">Melissa Linkert, Curtis T. Rueden, Chris Allan, Jean-Marie Burel, Will Moore, Andrew Patterson, Brian Loranger, Josh Moore, Carlos Neves, Donald MacDonald, Aleksandra Tarkowska, Caitlin Sticco, Emma Hill, Mike Rossner, Kevin W. Eliceiri, and Jason R. Swedlow (2010) Metadata matters: access to image data in the real world. The Journal of Cell Biology, Vol. 189no. 5777-782 doi: 10.1083/jcb.201004104</p>
```
Comment (hflynn) [inline citing-us/index.html]: Having pasted the content without checking the older entries thoroughly, this should be '...Journal of Cell Biology 189(5), 777-782.

File: citing-us/index.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        
+        <!-- begin citation--> 
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-12 medium-4 columns">
+                <h3>OME-XML &amp; OME-TIFF</h3>
+            </div>
+            <div class="small-12 medium-8 columns">
+                <p>The 2010 article outlining the importance of metadata and introducing the OME Compliant Specification:</p>
+                <p class="citation">Melissa Linkert, Curtis T. Rueden, Chris Allan, Jean-Marie Burel, Will Moore, Andrew Patterson, Brian Loranger, Josh Moore, Carlos Neves, Donald MacDonald, Aleksandra Tarkowska, Caitlin Sticco, Emma Hill, Mike Rossner, Kevin W. Eliceiri, and Jason R. Swedlow (2010) Metadata matters: access to image data in the real world. The Journal of Cell Biology, Vol. 189no. 5777-782 doi: 10.1083/jcb.201004104</p>
```
Comment (hflynn) [inline citing-us/index.html]: As below '... Journal of Cell Biology 189(5), 777-782.

File: citing-us/index.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        
+        <!-- begin citation--> 
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-12 medium-4 columns">
+                <h3>Image Data Resource (IDR)</h3>
+            </div>
+            <div class="small-12 medium-8 columns">
+                <p class="citation">Eleanor Williams, Josh Moore, Simon W. Li, Gabriella Rustici, Aleksandra Tarkowska, Anatole Chessel, Simone Leo, Bálint Antal, Richard K. Ferguson, Ugis Sarkans, Alvis Brazma, Rafael E. Carazo Salas, Jason R. Swedlow (2017) The Image Data Resource: A Bioimage Data Integration and Publication Platform. Nature Methods 14(8), 775-781. Published 19 June 2017</p>
+                <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536224/" target="_blank">PubMedCentral: PMC5536224</a></p>
```
Comment (hflynn) [inline citing-us/index.html]: Replace 'PubMedCentral:' with 'PMCID:'
(sorry wasn't sure if that was the convention but found an example now)

File: citing-us/index.html [hunk 6]
```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Citing OME
+---          
+        <div class="callout large primary" id="bg-image-licensing">
+            <div class="row column text-center">
+                <h1>Citing Us</h1>
+                <p class="hero-subheader small-12 medium-12">We are delighted to have you cite the work of the Open Microscopy Environment from your own publications.</p>
```
Comment (hflynn) [inline citing-us/index.html]: I wonder if we should update this text, maybe something along the lines of
"As a grant funded project, OME would be grateful if you would cite our work in your own publications."
what do you think @sbesson / @joshmoore ?

Comment (hflynn) [inline citing-us/index.html]: "Please cite our work in your own publications to help us secure grant funding for future development of OME products." ?

File: citing-us/index.html (hunk 2 above)
Comment (lunson) [inline citing-us/index.html]: The naming/menu title will ultimately be decided by Jason. We'll bring it up at the next meeting, and I'll change the files accordingly.

I'll work on the other edits below and push in a bit.

**Comment by hflynn** [comment]
👍 thanks for the edits @lunson I think this is just waiting on Jason clearing the title and header text now. https://snoopycrimecop.github.io/www.openmicroscopy.org/citing-us/ is updated

**Comment by hflynn** [comment]
Oh, except maybe we want to add a cross-reference to this page under the general section on the licensing page too? We could just use the same text as the subheader

**Comment by hflynn** [comment]
Jason suggests replacing the OME server with the original OME vision on https://trello.com/c/DR1L8nDA/100-add-citations but I can make that change on a following PR if that's easier

**Comment by hflynn** [comment]
Decision - Citing OME

**Comment by lunson** [comment]
@hflynn The "Citing OME" change was the only thing I was waiting on for this PR, so this one is all yours now for Jason's OME vision request. 👍

**Comment by hflynn** [comment]
Looks good, merging to implement the final citation change on a new PR

**Pull request merged**